UnicornRender/video/source/Primitives.cpp

```diff
@@ -61,13 +65,76 @@ void Primitives::Cube(Mesh& mesh)
 void Primitives::Quad(Mesh& mesh)
 {
     std::vector<Vertex> vertices{{
-        {{-1.0f , -1.0f , 0.0f}, {1.0f , 0.0f}} ,
-        {{1.0f , -1.0f , 0.0f}, {0.0f , 0.0f}} ,
-        {{1.0f , 1.0f , 0.0f}, {0.0f , 1.0f}},
-        {{-1.0f , 1.0f , 0.0f}, {1.0f , 1.0f}},
+        {{-0.5f , -0.5f , 0.0f},{1.0f , 0.0f}} ,
+        {{0.5f , -0.5f , 0.0f},{0.0f , 0.0f}} ,
+        {{0.5f , 0.5f , 0.0f},{0.0f , 1.0f}},
+        {{-0.5f , 0.5f , 0.0f},{1.0f , 1.0f}},
     }};
 
     mesh.SetMeshData(vertices, {0, 1, 2, 2, 3, 0});
 }
+
+void Primitives::Sphere(Mesh& mesh, float radius, uint32_t rings, uint32_t sectors)
+{
+    if(radius < 0)
+    {
+        LOG_WARNING("Sphere radius less than 0! UV will be inverted!");
+    }
+    if(rings < 4 || sectors < 4)
```

Comment:
This is not the check you've mentioned in the doxygen comment.
Update documentation that neither rings nor sectors can be less than 4 and remove part about `rings*sphere > 16` because it would be redundant.